src/analysis/history.cpp

```diff
@@ -279,4 +280,100 @@ void ReduceLocalizationFunction(MeshData<Real> *md) {
 
 } // exp
 
+using namespace parthenon::package::prelude;
+template <typename F>
+KOKKOS_INLINE_FUNCTION Real ComputeDivInPillbox(int ndim, int b, int k, int j, int i,
+                                                const parthenon::Coordinates_t &coords,
+                                                const F &f) {
+  Real div_mass_flux_integral;
+  div_mass_flux_integral =
+      -(f(b, 1, k, j, i + 1) - f(b, 1, k, j, i)) * coords.FaceArea<X1DIR>(k, j, i);
```

Comment:
Remove the minus signs here, and add them to the value in `mdot` as the divergence doesn't have a minus sign in its definition but `mdot` does, due to the orientation of surface normals.

Reply:
done